pkg/injector/patch_test.go

```diff
@@ -2,50 +2,134 @@ package injector
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"testing"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	mapset "github.com/deckarep/golang-set"
 	"github.com/golang/mock/gomock"
 	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
```

Comment:
Is this import still necessary? I don't see we using describe any more here.

Reply:
It's actually used here[ ](https://github.com/openservicemesh/osm/pull/3519/files#diff-3a53dad499d16baa10ff7e0fc76ceb1a7805ca8abcc83e308fef48c5ece1f45aR127) but I can remove the dependency 👍